packages/deployments/contracts/contracts/core/connext/libraries/LibDiamond.sol

```diff
@@ -155,7 +156,8 @@ library LibDiamond {
     if (selectorPosition == 0) {
       addFacet(ds, _facetAddress);
     }
-    for (uint256 selectorIndex; selectorIndex < _functionSelectors.length; selectorIndex++) {
+    uint256 len = _functionSelectors.length;
```

Comment:
If the `len` is calculated as the first line of the function then the following optimization can be done:
```solidity
require(len != 0, "LibDiamondCut: No selectors in facet to cut");
```